LithoUtils/Classes/Base/Alert/Alert.swift

```diff
@@ -10,40 +10,137 @@ import LithoOperators
 import Prelude
 import fuikit
 
+public func defaultAlertAction(title: String, handler: @escaping (UIAlertAction) -> Void) -> UIAlertAction {
+    return UIAlertAction(title: title, style: .default, handler: handler)
+}
+
+public func cancelAlertAction(title: String, handler: @escaping (UIAlertAction) -> Void) -> UIAlertAction {
+    return UIAlertAction(title: title, style: .cancel, handler: handler)
+}
+
 public func alertController(title: String?, message: String?) -> UIAlertController {
-    let alert = UIAlertController()
-    alert.title = title
-    alert.message = message
-    alert.addAction(UIAlertAction(title: "Okay", style: .default, handler: ignoreArg({ alert.dismiss(animated: true, completion: nil) })))
+    let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+    alert.addAction(defaultAlertAction(title: "Okay", handler: ignoreArg(nil *> alert.dismissAnimated)))
     return alert
 }
 
-public func dismissableAlert(title: String?, message: String?, actionTitle: String, onDismiss: @escaping () -> Void) -> UIAlertController {
-    let alert = alertController(title: title, message: message)
-    let action = UIAlertAction(title: actionTitle, style: .default, handler: ignoreArg(onDismiss *> alert.dismissClosure()))
-    alert.addAction(action)
+public func alertControllerSafe(title: String?, message: String?, barButton: UIBarButtonItem) -> UIAlertController {
+    return alertController(title: title, message: message).configure(barButton -*> popOverCompatibility)
+}
+
+public func alertControllerSafe(title: String?, message: String?, sourceView: UIView) -> UIAlertController {
+    return alertController(title: title, message: message).configure(sourceView -*> popOverCompatibility)
+}
+
+public func alertControllerSafe(title: String?, message: String?, sourceRect: CGRect) -> UIAlertController {
+    return alertController(title: title, message: message).configure(sourceRect -*> popOverCompatibility)
+}
+
+public func actionSheet(title: String?, message: String?) -> UIAlertController {
+    let alert = UIAlertController(title: title, message: message, preferredStyle: .actionSheet)
+    alert.addAction(defaultAlertAction(title: "Okay", handler: ignoreArg(nil *> alert.dismissAnimated)))
     return alert
 }
 
-public func dismissableAlert(title: String?, message: String?, actionTitle: String) -> UIAlertController {
+public func actionSheetSafe(title: String?, message: String?, barButton: UIBarButtonItem) -> UIAlertController {
```

Comment:
Instead of safe let's call it popoverSafe, so it's clear that that's why it's safe